XenAdmin/Controls/ComboBoxes/VgpuComboBox.cs

```diff
@@ -195,6 +195,9 @@ public bool Equals(GpuTuple other)
                     if (!VgpuTypes[i].Equals(other.VgpuTypes[i]))
                         return false;
                 }
+
+                if (IsGpuHeaderItem && other.IsVgpuSubitem)

```

Comment:
If we are changing this method, could we make sure other == null would not throw a null reference exception?